modules/database/test/ioc/db/dbStressLock.c

```diff
@@ -120,6 +120,9 @@ void doMulti(workerPriv *p)
     }
     dbScanUnlockMany(locker);
 
+    if (0)
+        testDiag("sum = %d", sum);
```

Comment:
Debugging leftovers?

Reply:
Core Group: Use the DEBUG macro in the `if ()`.